Python tests should not assert order of output

## Description

```
_________________________________ test_compose _________________________________

    def test_compose():
        con = glaredb.connect()
        con.execute("create table tbl (a int, b int);")
        con.execute("insert into tbl values (1, 2);")
        con.execute("insert into tbl values (3, 4);")
        con.execute("insert into tbl values (5, 6);")
        intermediate = con.sql("select * from tbl where a > 2;")
        out_1 = intermediate.to_arrow().to_pydict()
        expected = {"a": [3, 5], "b": [4, 6]}
>       assert out_1 == expected
E       AssertionError: assert {'a': [5, 3], 'b': [6, 4]} == {'a': [3, 5], 'b': [4, 6]}
E         Differing items:
E         {'b': [6, 4]} != {'b': [4, 6]}
E         {'a': [5, 3]} != {'a': [3, 5]}
E         Full diff:
E         - {'a': [3, 5], 'b': [4, 6]}
E         ?         ---          ---
E         + {'a': [5, 3], 'b': [6, 4]}
E         ?        +++          +++

tests/test_logical_plan.py:14: AssertionError
=========================== short test summary info ============================
FAILED tests/test_logical_plan.py::test_compose - AssertionError: assert {'a': [5, 3], 'b': [6, 4]} == {'a': [3, 5], 'b': [4, 6]}
  Differing items:
  {'b': [6, 4]} != {'b': [4, 6]}
  {'a': [5, 3]} != {'a': [3, 5]}
  Full diff:
  - {'a': [3, 5], 'b': [4, 6]}
  ?         ---          ---
  + {'a': [5, 3], 'b': [6, 4]}
  ?        +++          +++
```

--- a/py-glaredb/tests/test_logical_plan.py
+++ b/py-glaredb/tests/test_logical_plan.py
@@ -1,6 +1,11 @@
 import pytest
 import glaredb
 
+def pydict_sort(pd):
+    res = dict()
+    for key in sorted(pd):
+        res[key] = sorted(pd[key])
+    return res
 
 def test_compose():
     con = glaredb.connect()
@@ -9,10 +14,11 @@ def test_compose():
     con.execute("insert into tbl values (3, 4);")
     con.execute("insert into tbl values (5, 6);")
     intermediate = con.sql("select * from tbl where a > 2;")
-    out_1 = intermediate.to_arrow().to_pydict()
+    out_1 = pydict_sort(intermediate.to_arrow().to_pydict())
     expected = {"a": [3, 5], "b": [4, 6]}
     assert out_1 == expected
     out_2 = con.sql("select * from intermediate where b > 4;").to_arrow().to_pydict()
     expected = {"a": [5], "b": [6]}
     assert out_2 == expected
     con.close()
+    

--- a/py-glaredb/tests/test_logical_plan.py
+++ b/py-glaredb/tests/test_logical_plan.py
@@ -1,11 +1,6 @@
 import pytest
 import glaredb
 
-def pydict_sort(pd):
-    res = dict()
-    for key in sorted(pd):
-        res[key] = sorted(pd[key])
-    return res
 
 def test_compose():
     con = glaredb.connect()
@@ -13,12 +8,11 @@ def test_compose():
     con.execute("insert into tbl values (1, 2);")
     con.execute("insert into tbl values (3, 4);")
     con.execute("insert into tbl values (5, 6);")
-    intermediate = con.sql("select * from tbl where a > 2;")
-    out_1 = pydict_sort(intermediate.to_arrow().to_pydict())
+    intermediate = con.sql("select * from tbl where a > 2 order by a;")
+    out_1 = intermediate.to_arrow().to_pydict()
     expected = {"a": [3, 5], "b": [4, 6]}
     assert out_1 == expected
     out_2 = con.sql("select * from intermediate where b > 4;").to_arrow().to_pydict()
     expected = {"a": [5], "b": [6]}
     assert out_2 == expected
     con.close()
-    

